dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/singleevent/SingleEventChangeLog.java

```diff
@@ -58,7 +58,7 @@
 public class SingleEventChangeLog {
   @Id
   @GeneratedValue(strategy = GenerationType.SEQUENCE)
-  @SequenceGenerator(name = "eventchangelog_sequence")
+  @SequenceGenerator(sequenceName = "singleeventchangelog_sequence")
```

Comment:
Did you check what the common pattern is? `GeneratedValue`  needs to be linked to the `SequenceGenerator` I think. If we don't do it explicitly its being looked up but I don't know how and what it would fallback to 🤷🏻 maybe implicit is ok for us but maybe not

```java
@SequenceGenerator(
    name = "singleeventchangelog_gen",
    sequenceName = "singleeventchangelog_sequence"
)
@GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "singleeventchangelog_gen")
```